docs/src/quickstart.md

```diff
@@ -160,6 +161,63 @@ plot_energy(idata);
 gcf()
 ```
 
+### Additional information in Turing.jl
+With a few more steps, we can use Turing to compute additional useful groups to add to the [`InferenceData`].(@ref).
+
+To sample from the prior, one simply calls `sample` but with the `Prior` sampler:
+```@example quickstart
+prior = sample(param_mod, Prior(), nsamples; progress = false)
+```
+To draw from the prior and posterior predictive distributions we can instantiate a "predictive model", i.e. a Turing model but with the observations set to `missing`, and then calling `predict` on the predictive model and the previously drawn samples:
+```@example quickstart
+# Instantiate the predictive model
+param_mod_predict = turing_model(J, similar(y, Missing), σ)
+# and then sample!
+prior_predictive = predict(param_mod_predict, prior)
+posterior_predictive = predict(param_mod_predict, turing_chns)
+```
+And to extract the elementwise log-likelihoods, which is useful if you want to compute metrics such as [`loo`](@ref),
+```@example quickstart
+loglikelihoods = Turing.elementwise_loglikelihoods(param_mod, turing_chns)
+```
+This can then be included in the `from_mcmcchains` call from above:
+```@example quickstart
+# Convert into format compatible with ArviZ.jl
+using LinearAlgebra
+# Ensure the ordering of the loglikelihoods matches the ordering of `posterior_predictive`
+loglikelihoods_vals = getindex.(Ref(loglikelihoods), string.(keys(posterior_predictive)))
+# Reshape into `(num_chains, num_samples, num_params)`
```

Comment:
Would it be helpful here to use the variable names defined in this page?
```suggestion
# Reshape into `(nchains, nsamples, size(y)...)`
```

Reply:
That's up to you :+1: I think I maybe prefer `num_params` since it indicates what to do in general, while the code tells you how this particular example looks.

Also, it should probably be
```suggestion
# Reshape into `(num_chains, num_samples, length(y))`
```
instead?

Reply:
Well if we were going with completely general, ArviZ uses `(nchain, ndraw, shape...)`. We here `size` instead of `shape` (because `np.shape` is the same as `size`). So the version I gave above is closer to the general version, and I'd prefer it. 

But I also should go through and replace `nchains` with `nchain` and `nsamples` with `ndraw` in a later PR.

Reply:
Ah, aight. I'll let you choose this one :+1:  
Hmm, yeah so it's actually slightly annoying to recover the correct shape from `MCMCChains.Chains` AFAIK. That's why I just went with the elementwise one.

Reply:
Yeah, makes sense, the [code we have](https://github.com/arviz-devs/ArviZ.jl/blob/master/src/mcmcchains.jl#L32-L40) for inferring size from `Chains` relies on the indices and supports the indexing conventions of CmdStan and Turing (last I checked CmdStan uses dots instead of brackets to indicate indexing), but it could probably be improved. I'd rather not make that functionality part of the API, so I think you did the right thing here.

What we could always do is add a `from_turing` method consistent with the other `from_XXX` methods that forwards everything to `from_mcmcchains` but first pre-processes `log_likelihood`. That's the cleanest solution I can think of right now.